test/unit/transpiler/passes/scheduling/test_scheduler.py

```diff
@@ -870,49 +868,6 @@ def test_registers(self):
 
         self.assertEqual(expected, scheduled)
 
-    def test_c_if_plugin_conversion_with_transpile(self):
```

Comment:
Looks like the `c_if` control flow is no longer supported so I think we can remove these tests? 

```
'c_if control-flow is not supported by this pass. Please apply "ConvertConditionsToIfOps" to convert these conditional operations to new-style Qiskit control-flow.
```